gnomad_qc/v3/sample_qc/sample_qc.py

```diff
@@ -534,6 +631,17 @@ def assign_pops(
     if pop is not None:
         meta_ht = meta.ht()
         pop_pca_scores_ht = pop_pca_scores_ht.annotate(**meta_ht[pop_pca_scores_ht.key])
+        # Set subpop_descritpion to missing if the subpop is not in the manually curated list of subpops for the given pop
+        pop_pca_scores_ht = pop_pca_scores_ht.annotate(
+            project_meta=pop_pca_scores_ht.project_meta.annotate(
+                subpop_description=hl.or_missing(
+                    hl.literal(CURATED_SUBPOPS[pop]).contains(
+                        pop_pca_scores_ht.project_meta.subpop_description
+                    ),
+                    pop_pca_scores_ht.project_meta.subpop_description,
+                )
+            )
+        )
```

Comment:
Adding comment here because I can't add below, but just noticed it. Does this work if not restricting to a population? 

```
pop_pca_scores_ht = pop_pca_scores_ht.annotate(
        withheld_sample=hl.is_defined(pop_pca_scores_ht.project_meta.subpop_description)
        & (~hl.is_defined(pop_pca_scores_ht.training_pop))
    )
```